internal/daemon/controller/handlers/hosts/host_service.go

```diff
@@ -643,15 +643,16 @@ func validateCreateRequest(req *pbs.CreateHostRequest) error {
 					len(attrs.GetAddress().GetValue()) < static.MinHostAddressLength ||
 					len(attrs.GetAddress().GetValue()) > static.MaxHostAddressLength {
 					badFields["attributes.address"] = fmt.Sprintf("Address length must be between %d and %d characters.", static.MinHostAddressLength, static.MaxHostAddressLength)
-				}
-				_, _, err := net.SplitHostPort(attrs.GetAddress().GetValue())
-				switch {
-				case err == nil:
-					badFields["attributes.address"] = "Address for static hosts does not support a port."
-				case strings.Contains(err.Error(), "missing port in address"):
-					// Bare hostname, which we want
-				default:
-					badFields["attributes.address"] = fmt.Sprintf("Error parsing address: %v.", err)
+				} else {
+					_, _, err := net.SplitHostPort(attrs.GetAddress().GetValue())
+					switch {
+					case err == nil:
+						badFields["attributes.address"] = "Address for static hosts does not support a port."
```

Comment:
perhaps we can define a const for `attributes.address` since it's referenced 5 times in this small change.  Might be a good idea to define one from `attributes` as well, but that's a bigger change.

Reply:
✅

Reply:
I changed "attributes" in a few places (we already had a globals), but a future PR can just sweep through and update.

We may also want to consider moving things out of globals into a fields subpackge, so instead of e.g. "globals.AttributesField" we can just have "fields.Attributes". Maybe.